tokens/src/lib.rs

```diff
@@ -278,6 +297,12 @@ pub mod module {
 			free: T::Balance,
 			reserved: T::Balance,
 		},
+		/// Some amount was removed from the account (e.g. for misbehavior).
+		Slashed {
+			currency_id: T::CurrencyId,
+			who: T::AccountId,
+			amount: T::Balance,
+		},
```

Comment:
would be conflict with `Slashed` event in https://github.com/open-web3-stack/open-runtime-module-library/pull/729